PocketKit/Sources/PocketKit/Item/ItemPresenter.swift

```diff
@@ -19,6 +19,12 @@ class ItemPresenter: ItemRow {
     public var title: String {
         item.title ?? item.thumbnailURL?.absoluteString ?? "Missing Title"
     }
+    
+    public var languageDirection: LanguageDirection {
+        let language = item.language ?? "en"
```

Comment:
The majority of articles returned are "en", so I think it's safe to default to "en" if no language is returned from the server.

Reply:
The majority of articles are "en", sure. But that is probably because the majority of our users are "en" speakers.

For speakers of RTL languages, what language are the majority of articles in? Put more broadly, for any user are the majority of articles in that user's system language? Or are the majority of articles still "en", even if their system language is non-en?